manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.locale.en-US.yaml

```diff
@@ -0,0 +1,25 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JetBrains.DataSpell
+PackageVersion: 2022.3.2
+PackageLocale: en-US
+Publisher: JetBrains s.r.o.
+PublisherUrl: https://www.jetbrains.com
+PublisherSupportUrl: https://www.jetbrains.com/data-tools
+PrivacyUrl: https://www.jetbrains.com/privacy-security
+PackageName: JetBrains DataSpell
+PackageUrl: https://www.jetbrains.com/dataspell
+License: Proprietary
+Copyright: Copyright (c) JetBrains s.r.o.
+CopyrightUrl: https://www.jetbrains.com/legal/#licensing
+ShortDescription: A new IDE by JetBrains for data science.
+Moniker: dataspell-preview
```

Comment:
Is the package still in Preview? If so, then consider adding it under [JetBrains.DataSpell.EarlyPreview](https://github.com/microsoft/winget-pkgs/tree/master/manifests/j/JetBrains/DataSpell/EarlyPreview) where other versions already exist.

Reply:
No, sorry, I just copied the manifest for the preview to add the non-preview. Have fixed